internal/supervisor/import.go

```diff
@@ -44,90 +49,293 @@ func NewImportSupervisor(store importStore, logger log.FieldLogger, cloudClient
 
 func (s *ImportSupervisor) Start() {
 	s.logger.Info("Import supervisor started")
-	go func() {
-		for {
-			s.supervise()
-			time.Sleep(30 * time.Second) // TODO: make this configurable
-		}
-	}()
+
+	tick := time.NewTicker(time.Second * 30)
+
+	for range tick.C {
+		s.do()
+	}
 }
 
-func (s *ImportSupervisor) supervise() {
-	imports, err := s.store.GetImportsInProgress()
+func (s *ImportSupervisor) do() {
+	imports, err := s.store.GetUnlockedImportPendingWork()
 	if err != nil {
-		s.logger.WithError(err).Error("Failed to look up ongoing Imports to supervise")
+		s.logger.WithError(err).Errorln("Failed to query for import pending work")
```

Comment:
Any reason why you started using the `ln` type loggers? Where the logs not showing up on new lines already?

Reply:
Not sure why. But you are right. New logs are always in new lines.